### DISCUSSION TITLE
When i create a Second Site i get a Failure if i try  to file a report

### DISCUSSION THREAD

**Discussion by eikman2k** [globaleaks/globaleaks-whistleblowing-software#3663] [Q&A] [answered]
Installed several times completly!
The First Site works correctly but the Second is not able to ge a File Report! (Correction my first Installation Worked, now i installed a newer Version, and now it is not possible to report a file)

i alway get the Message: "unable to deliver the submission to least one recipient"

i createt one Admin and one Recipient (both activatet) but i can not report a File

System is a blank Linux Debian Bookworm 12

PLS Help.

**Comment by MichalRuz** [comment]
I have the same problem.
Could it be an unset ssl certificate (https protocol)?

**Reply by evilaliv3** [reply]
No the ssl certificate do not have impact on this issue.

**Comment by evilaliv3** [comment, marked as answer]
This is related to the fact that you have not created a recipient or you have not assigned a recipient to the reporting context.

**Reply by MichalRuz** [reply]
Yes, thank you.
